LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2017, dask-ndmeasure Developers (see AUTHORS.rst for details)
```

Comment:
Using `dask-ndmeasure` here was a typo. Should have been `dask-distance`. Fixing in PR ( https://github.com/jakirkham/dask-distance/pull/93 ).